utils/forest_shim/src/clock.rs

```diff
@@ -1,8 +1,27 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-pub use fvm_shared::clock::ChainEpoch;
-use fvm_shared::clock::EPOCH_DURATION_SECONDS;
-
 pub const SECONDS_IN_DAY: i64 = 86400;
 pub const EPOCHS_IN_DAY: i64 = SECONDS_IN_DAY / EPOCH_DURATION_SECONDS;
+pub const EPOCH_DURATION_SECONDS: i64 = 30;
```

Comment:
We should not use this constant at all because it's valid only for mainnet and calibnet. But that's to be addressed elsewhere.